tests/dialects/test_snowflake.py

```diff
@@ -976,12 +976,17 @@ def test_snowflake(self):
                 "snowflake": "EDITDISTANCE(col1, col2, 3)",
             },
         )
-        self.validate_identity("SELECT BITOR(a, b) FROM table")
+        self.validate_identity("SELECT BITOR(a, b)")
 
-        self.validate_identity("SELECT BIT_OR(a, b) FROM table", "SELECT BITOR(a, b) FROM table")
+        self.validate_identity("SELECT BIT_OR(a, b)", "SELECT BITOR(a, b)")
 
-        # Test BITOR with three arguments, padding on the left
-        self.validate_identity("SELECT BITOR(a, b, 'LEFT') FROM table_name")
+        self.validate_identity("SELECT BITOR(a, b, 'LEFT')")
+
+        self.validate_identity("SELECT BITXOR(a, b, 'LEFT')")
+
+        self.validate_identity("SELECT BITSHIFTLEFT(a, 1)")
+
+        self.validate_identity("SELECT BITSHIFTRIGHT(a, 1)")
```

Comment:
Nit: let's remove the whitespaces between each `validate_identity` call to reduce file size